pmoves/services/creator-operator/tests/test_voice_operator.py

```
@@ -0,0 +1,49 @@
+
+def test_run_voice_error_path(tmp_path):
+    wo = {"workorder_id": "wo_v2", "workflow_id": "voice.omnivoice",
+          "knobs": {"text": "", "voice_ref": "bean"},
+          "license_ack": {"model": "omnivoice", "mode": "local", "ack": True}}
+    client = FakeOmniVoiceClient(out_dir=tmp_path)
+    r = run_voice(wo, client)
+    assert r["status"] == "error" and r["artifact"] is None and r["error"]
+
+
+class _FakeSinks:
+    def __init__(self): self.nats=[]; self.notebook=[]; self.discord=[]; self.n8n=[]
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)